src/utils.js

```diff
@@ -75,3 +75,32 @@ export function createTypeStrategy(type) {
     );
   };
 }
+
```

Comment:
The following function is very similar to other implementations such as `lodash:debounce`. I just didn't want to add another dependency to the project for a single function call.